recipes/qxmpp/all/conanfile.py

```diff
@@ -46,13 +46,13 @@ def configure(self):
             del self.options.fPIC
 
     def requirements(self):
-        self.requires("qt/6.2.3")
+        self.requires("qt/6.2.4")
         if self.options.with_gstreamer:
             self.requires("gstreamer/1.19.2")
             self.requires("glib/2.70.1")
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+        tools.files.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
```

Comment:
https://docs.conan.io/en/latest/reference/conanfile/tools.html#conan-tools

tools.xxx.yyy can not be used. You have two options:

- from conan.tools.files import yyy
- from conan.tools import xxx